### PR TITLE
DP-7094 Dataportaal frontpage assembly

### PR DESCRIPTION
**Pull request by aditudorache**
## Amsterdam styled components pull request

Before opening a pull request, please ensure:

- [ ] The default export from a file matches the file name
- [ ] You have been following the guidelines, written in the [README](../docs/CONTRIBUTING.md#user-content-conventions-and-rules) file
- [ ] Your code has the necessary tests written
- [ ] You have updated the [CHANGELOG.md unreleased sections](../CHANGELOG.md#muser-content-unreleased)

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: packages/asc-ui/src/components/Body/Body.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
Comment (royderks) [inline packages/asc-ui/src/components/Body/Body.tsx]: Don't think this should be in here, as this is very specific. Have a look at the `atlas` repository where a container component specifically for that project is used

File: packages/asc-ui/src/components/Page/examples/DataportaalFrontpage.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import React from 'react'
```
Comment (royderks) [inline packages/asc-ui/src/components/Page/examples/DataportaalFrontpage.stories.tsx]: Would move this to the `atlas` repository as it's too specific

File: packages/asc-ui/src/components/Body/Body.tsx (hunk 1 above)
Comment (chrisvanmook) [reply packages/asc-ui/src/components/Body/Body.tsx]: I agree. Let's have a look what might be missing in the grid as well perhaps?

File: packages/asc-ui/src/components/shared/sizes.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const HEADER_MAX_WIDTH = 1600
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/shared/sizes.ts]: these sizes are specific for atlas, perhaps a good time to refactor now? They shouldn't be in asc..

File: packages/asc-ui/src/components/Header/examples/DataportaalHeader.stories.tsx [hunk 4]
```diff
@@ -78,7 +91,52 @@ const DataportaalHeader: React.FC<{}> = () => (
...
+  ))
+  .add('implementation for dataportaal - Frontpage', () => (
+    <>
+      <DataportaalHeader tall fullWidth={false} />
+      <DataportaalBody tall fullWidth={false}>
+        <BodyFiller />
+      </DataportaalBody>
+    </>
+  ))
+  .add('implementation for dataportaal - Editorial content', () => (
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Header/examples/DataportaalHeader.stories.tsx]: I think one example is enough... we just want to show an example of how we implement the header in a project.

File: packages/asc-ui/src/components/Header/examples/BodyFiller.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import Heading from '../../Heading'
+import Typography from '../../Typography'
+
+const BodyFiller: React.FC = () => (
```
Comment (chrisvanmook) [inline packages/asc-ui/src/components/Header/examples/BodyFiller.stories.tsx]: IMO plz (re)move this... This isn't part of a Header, plus do we need to show other stuff besides the Header in a story? Causes much confusion I think. Stories are becoming too complex

File: packages/asc-ui/src/components/Body/Body.tsx (hunk 1 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Body/Body.tsx]: We need to soort this out. The current version of the Header is also too specific so we need a Body component if we want to cover all the Header state scenario's. Perhaps a basic header and an atlas specific Header+Body component will adress all the needs.

File: packages/asc-ui/src/components/Page/examples/DataportaalFrontpage.stories.tsx (hunk 2 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Page/examples/DataportaalFrontpage.stories.tsx]: This is the development version, when is ready, we can move it to atlas

File: packages/asc-ui/src/components/shared/sizes.ts (hunk 3 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/shared/sizes.ts]: These are indeed the implementation for atlas. We'll look into it.

File: packages/asc-ui/src/components/Header/examples/DataportaalHeader.stories.tsx (hunk 4 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Header/examples/DataportaalHeader.stories.tsx]: Not really agree, the different states of a header (even though atlas specific) should be pictured in the examples, is the best way to document this.

File: packages/asc-ui/src/components/Header/examples/BodyFiller.stories.tsx (hunk 5 above)
Comment (aditudorache) [reply packages/asc-ui/src/components/Header/examples/BodyFiller.stories.tsx]: THanks, is just for development. Maybe moving the complexer specifi stories to a separate folder is not a bad idea

File: packages/asc-ui/src/components/Footer/Footer.stories.tsx [hunk 6]
```diff
@@ -102,86 +102,89 @@ const HelpLinks: React.FC<{ indent?: boolean }> = () => (
...
+      </Row>
+    </FooterBottom>
+  </Footer>
+)
+
+storiesOf('Composed/Footer', module)
+  .addDecorator(storyFn => (
+    <div style={{ padding: '40px 10px' }}>{storyFn()}</div>
   ))
+  .add('implementation for dataportaal', () => <DataportaalFooter />)
```
Comment (royderks) [inline packages/asc-ui/src/components/Footer/Footer.stories.tsx]: Shouldn't this be deleted then?

Comment (aditudorache) [reply packages/asc-ui/src/components/Footer/Footer.stories.tsx]: out of scope

File: packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+          I don&apos;t have the CompactThemeProvider applyed.
+        </Heading>
+        <Paragraph>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Debitis
+          deleniti dicta doloremque eveniet explicabo id ipsum omnis placeat
+          rerum suscipit.
+        </Paragraph>
+      </div>
+      <div>
+        <CompactThemeProvider>
```
Comment (chrisvanmook) [inline packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx]: This is already in the Paragraph story isn't it?

File: packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx [hunk 8]
```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import Paragraph from '../components/Paragraph'
+import Heading from '../components/Heading'
+import CompactThemeProvider from './CompactThemeProvider'
+
+storiesOf('Atoms/Typography/CompactThemeProvider', module).add(
```
Comment (chrisvanmook) [inline packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx]: CompactThemeProvider isn't really a component / part of Typography... Consider deleting this?

File: packages/asc-ui/src/components/Footer/Footer.stories.tsx [hunk 9]
```diff
@@ -102,86 +102,89 @@ const HelpLinks: React.FC<{ indent?: boolean }> = () => (
...
-            <>
-              <FooterToggle title="Colofon" hideAt="tabletM">
-                <FooterContent indent>
-                  <ColofonLinks />
-                </FooterContent>
-              </FooterToggle>
-              <FooterContent showAt="tabletM">
-                <FooterHeading $as="h3">Colofon</FooterHeading>
+// eslint-disable-next-line import/prefer-default-export
+export const DataportaalFooter: React.FC<{}> = () => (
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply packages/asc-ui/src/components/Footer/Footer.stories.tsx]: out of scope for this pr, will make a new issue, to move the rest of the specific atlas implementations

File: packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts [hunk 10]
```diff
@@ -3,19 +3,19 @@ import {
   breakpoint,
   themeColor,
   customCss,
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts]: out of scope

File: packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts [hunk 11]
```diff
@@ -36,10 +36,9 @@ const HeaderWrapperStyle = styled.div<Props>`
     fullWidth ? 'none' : `${WRAPPER_MAX_WIDTH}px`};
   box-shadow: 0 4px rgba(0, 0, 0, 0.04);
   box-sizing: content-box;
-  position: fixed; /* Unfortunately no sticky because IE11 */
-  left: 0;
-  right: 0;
-  z-index: 2; /* Orders the header higher than the content */
+  position: sticky;
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts]: out of scope of this pr, will address it  as a part of the integration of the landing page in atlas

File: packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx (hunk 8 above)
Comment (aditudorache) [reply packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx]: ok

File: packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx (hunk 7 above)
Comment (aditudorache) [reply packages/asc-ui/src/theme/CompactThemeProvider.stories.tsx]: will delete the whole story then

File: CHANGELOG.md [hunk 12]
```diff
@@ -15,9 +15,13 @@ Prefix the change with one of these keywords:
 - _Security_: in case of vulnerabilities.
 
 ## UNRELEASED
+
+## [0.14.1] - 2019-09-05
```
[the remark this answers is not in the document]
Comment (aditudorache) [reply CHANGELOG.md]: version removed for now

File: packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts (hunk 11 above)
[the remark this answers is not in the document]
Comment (chrisvanmook) [reply packages/asc-ui/src/components/Header/HeaderWrapperStyle.ts]: not agreed this is out of scope, it's a breaking change / introducing a bug for the header.

**Pull request merged**